### PR TITLE
Gianni review 2

### PR DESCRIPTION
**Pull request by giannidallatorre**
# Summary 

----

**Related issue :**

### CODE REVIEW

**Review by gwarf — CHANGES_REQUESTED** [review]
Thanks, looks good, please just update your branch with the latest changes on master.

**Comment by gwarf** [comment]
I've tried the web resolve-commits tool, let's see

**Review by gwarf — APPROVED** [review]

File: content/en/users/cloud-compute/auth/_index.md [hunk 1]
```diff
@@ -74,7 +83,42 @@ approved you will be able to interact with the infrastructure.
 
 Pre-existing VOs of EGI can be also used on IaaS cloud providers.
 Consult with your VO manager or browse the existing VOs at the [EGI
-Operations Portal](https://operations-portal.egi.eu/vo/).
+Operations Portal](https://operations-portal.egi.eu/vo/). Joining these
+VOs may require a personal X.509 certificate. For getting this certificate,
```
Comment (enolfc) [inline content/en/users/cloud-compute/auth/_index.md:87]: This info is now in Check-in VOMS (../check-in/voms)

**Review by enolfc — COMMENTED** [review]
Thanks @giannidallatorre. Most changes are good, but there were some reorg of the files that should be taken into account.

**Review by andrea-manzi — DISMISSED** [review]

File: content/en/users/cloud-compute/auth/_index.md (hunk 1 above)
Comment (giannidallatorre) [reply content/en/users/cloud-compute/auth/_index.md:87]: Do we want to remove the part from:
`For getting this certificate` and just link to:
`../voms/#get-a-certificate` ?

**Review by gwarf — APPROVED** [review]
LGTM.

File: content/en/users/cloud-compute/auth/_index.md [hunk 2]
```diff
@@ -74,7 +83,38 @@ approved you will be able to interact with the infrastructure.
...
+1.  Obtain a valid OAuth2.0 access token from Check-in. Access tokens
+    are short-lived credentials that can be obtained by recognised
+    Check-in clients once a user has been authenticated.
+1.  Interchange the Check-in access token for a valid unscoped Keystone
+    token.
+1.  Discover available projects from Keystone using the unscoped token.
+1.  Use the unscoped Keystone token to get a scoped token for a valid
+    project. Scoped tokens will allow the user to perform operations on
+    the provider.
+
```
Comment (enolfc) [inline content/en/users/cloud-compute/auth/_index.md:117]:
```suggestion
```

This content is already above (line 10 onwards)

File: content/en/users/cloud-compute/auth/_index.md [hunk 3]
```diff
@@ -35,7 +35,16 @@ The process for authentication is as follows:
...
 Cloud Compute service is accessed through **Virtual Organisations
-(VOs)**. Users that are members of a VO will have access to the providers
+(VOs)**. A VO is a grouping of IaaS cloud providers from the federation,
+who allocate capacity for a specific user group. Users with similar
+interestes/requirements can join or form a VO to gather resources from EGI
+cloud providers - typically for a given project, experiment or use case.
+There are generic VOs too, for example the vo.access.egi.eu VO, which is
+open for any user who wants to experiment with the service. **You have
+to join a VO before you can interact with the cloud resources**, while
+higher level services (PaaS, SaaS) do not always require VO membership.
```
Comment (enolfc) [inline content/en/users/cloud-compute/auth/_index.md:45]: this is kind of duplication from check-in/signup, no need for it here I think

**Review by enolfc — CHANGES_REQUESTED** [review]

File: content/en/users/cloud-compute/auth/_index.md (hunk 2 above)
Comment (giannidallatorre) [reply content/en/users/cloud-compute/auth/_index.md]: 54c552a33413b569364d4f6111378e1384fbff22

**Review by giannidallatorre — COMMENTED** [review]
Just removed some parts probably appeared from a merge.

**Review by enolfc — APPROVED** [review]

**Pull request merged**